app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.dazn_movie_player
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+
+class SplashScreen : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        supportActionBar?.hide()
+
+        val handler = Handler(Looper.getMainLooper())
+        handler.postDelayed({
```

Comment:
Remove handler here and maintain in single line Handler(Looper.getMainLooper()).postDelayed({})